components/profileLink/index.test.jsx

```diff
@@ -48,7 +51,12 @@ describe("ProfileLink", () => {
       route: "/contacts",
     });
   });
-  it("supports rendering name from vcard.fn", () => {
+  it("renders first name from profile names array", () => {
```

Comment:
make sure you've a test for `mockedUseFullProfile.mockReturnValue(null)` as that's what the hook initially returns on first render.